dashboard/pages/home.py

```diff
@@ -12,9 +12,15 @@ def show_home():
     conn = database.get_connection()
 
     st.title("Home")
-    st.header("Welcome to the Bandcamp analysis tracker!")
-    st.write("""Here you'll find insights into Bandcamp sales
-             and you can also subscribe to receive email notifications!""")
+    col = st.columns(2)
+    with col[0]:
+        st.image('Apollo.png', use_column_width="auto")
+    with col[1]:
+        st.header("Welcome to Apollo!")
+        st.subheader("The Bandcamp Sales Tracker")
+        st.write(
+            "Here you'll find insights into Bandcamp sales and you can also subscribe to receive email notifications!")
```

Comment:
Is this too long for pylint? Can't be more than 100 characters.

Reply:
You're right. I've improved the pylint score by changing this